client/app/components/username.tsx

```diff
@@ -17,6 +17,10 @@ function Username({ showVerfied, user }: Props) {
 			{user.username === "notgr" && (
 				<div className="inline-block i-lucide-crown text-amber-500" />
 			)}
+			{/* This should be for moderators */}
+			{user.role === "moderator" && user.username !== "notgr" && (
```

Comment:
oh I'm moderator... I want some of the zap

Reply:
So you get a crown and a zap? I'm not sure you'll like the color combo tho. You suggest the change😅